docs/modules/control/move_to_a_pose_control/move_to_a_pose_control.rst

```diff
@@ -1,11 +1,102 @@
 Move to a pose control
 ----------------------
 
-This is a simulation of moving to a pose control
+This is a simulation of moving to a pose control.
 
 .. image:: https://github.com/AtsushiSakai/PythonRoboticsGifs/raw/master/PathTracking/move_to_pose/animation.gif
 
-Ref:
 
+PathFinderController class
```

Comment:
[General comment] I think the rst doc should focus on the mathematics and the algorithm of the example.  Documentation of codes should be in python script as docstrings or header comments. If you have time, please consider to fix the doc in the next PR.

Reply:
@AtsushiSakai  Thanks for your comments. I agree to keep the rst doc only for the mathematical explanations. Also, it will be great if we can automatically generate a Reference document based on the docstrings comments. I will try to address  your points in my next PR. 

Merry Christmas and Happy new Year :)